### PR TITLE
Rs 2022 07 04.2

### PR DESCRIPTION
**Pull request by rschlaikjer**

### CODE REVIEW

File: script/Arcueid/Day 5/05_00_ARC05_9.txt [hunk 1]
```diff
@@ -141,17 +141,25 @@ My consciousness turns to my rear, telling me to run. There's nothing I can do t
...
 -- だってしょうがない。アレを相手に何が出来る。あの女が。ここまでで十分だ。ここから先は過剰すぎる。あの女が。あとは逃げてしまえばいい。目を背けてしまえばいい。それが<ま|・><っ|・><と|・><う|・><な|・><人|・><間|・>だ。あの女が、殺される。
-It's no use after all. What could I possibly do against an opponent like that? That woman. From here on, I'm totally out of my depth. That woman. I should just run away. Just look away. Any %{i}decent human%{/i} would. That woman―――will be killed.
+It's no use after all.
+ What could I possibly do against an opponent like that?
+ That woman.
+ From here on, I'm totally out of my depth.
+ That woman.
+ I should just run away.
+ Just look away.
+ Any %{i}decent human%{/i} would.
```
Comment (Avuxo) [inline script/Arcueid/Day 5/05_00_ARC05_9.txt:151]: maybe italics just on "decent"?

File: script/Arcueid/Day 5/05_00_ARC05_9.txt [hunk 2]
```diff
@@ -525,18 +532,20 @@ I had three minutes in total before my body and consciousness to shut down due t
 [sha:9a6561de7b3b8e6f76e04f645fe9983f8576ae7a]{
 -- Page 591, Offset 10208.
 -- 満足に活動できるリミットは１分もない。
-I don't even have one minute left.
+// 05_00_ARC05_9B has him saying fairly explicitly that he has 2 min left,
+// so is this actually 'one full minute has already become nai'?
```
Comment (Avuxo) [inline script/Arcueid/Day 5/05_00_ARC05_9.txt:536]: I left a comment about that in 5_9B but nobody responded and I forgot to follow-up. I'm gonna _guess_ he's saying "I've already lost one minute" but that timeline doesn't line up either? I guess either way it makes _more_ sense this way

File: script/Arcueid/Day 5/05_00_ARC05_9_2B.txt [hunk 3]
```diff
@@ -494,7 +494,7 @@ The gliding descent begins.
 -- Page 688, Offset 10514.
 -- 死に迫る、一呼吸分のスライダー。
 // the punctuation in jp is abnormal but I'm kinda in favor of ignoring.
-A short slide一toward death itself.
+A short slide―――toward death itself.
```
Comment (Avuxo) [inline script/Arcueid/Day 5/05_00_ARC05_9_2B.txt:497]: image

File: script/Arcueid/Day 5/05_00_ARC05_9_3.txt [hunk 4]
```diff
@@ -198,7 +198,7 @@ A massive foreign object pierces my chest.
 [sha:3d297f21df4cf3d88728881691bd859a19fb03c7]{
 -- Page 814, Offset 10869.
 -- 俺は大きく、人生最後になる息を吐いて、
-I take a deep breath for the last time.
+I take a deep, final breath.
```
Comment (Avuxo) [inline script/Arcueid/Day 5/05_00_ARC05_9_3.txt:201]: prefer previous version.

**Review by Avuxo — APPROVED** [review]

File: script/Arcueid/Day 5/05_00_ARC05_9.txt (hunk 2 above)
Comment (rschlaikjer) [reply script/Arcueid/Day 5/05_00_ARC05_9.txt:536]: Yeah the time seems fucky, but this is at least 3 minutes -> down one minute -> 2 minutes left, even if there's a decent bit of file between (3-1) minutes and (2) minutes

Comment (Avuxo) [reply script/Arcueid/Day 5/05_00_ARC05_9.txt:536]: makes sense

**Pull request merged**